src/components/shared/NavBar.tsx

```diff
@@ -108,66 +107,85 @@ const NavBar: FC<NavBarProps> = ({}) => {
+            onClick={toggleDarkMode}
           />
 
-
-          <div className={`${isDarkMode ?  "text-white" : "text-secondary-500"}${
-            !isButtonClicked
-            ? " border-[1px] border-black hover:bg-gray-200 hover:border-gray-200"
-            : " border-[1px] border-secondary-500 hover:bg-black"
+          <div
+            className={`${isDarkMode ? "text-white" : "text-secondary-500"}${
+              !isButtonClicked
+                ? " border-[1px] border-black hover:bg-gray-200 hover:border-gray-200"
+                : " border-[1px] border-secondary-500 hover:bg-black"
             } py-2 px-2 rounded-[3px] max-md:px-1 max-md:py-1 flex items-center justify-center flex-col
             transition duration-300 ease-in-out relative
-            `} onClick={handleLanguageClick}>
+            `}
+            onClick={handleLanguageClick}
+          >
             <FaGlobe
-               className={`${
+              className={`${
                 !isButtonClicked ? "text-secondary-500" : "text-white"
-                } ${isDarkMode ?  "text-white" : "text-secondary-500" }  z-4 object-contain py-1 w-[29px] text-[32px]`}
+              } ${
+                isDarkMode ? "text-white" : "text-secondary-500"
+              }  z-4 object-contain py-1 w-[29px] text-[32px]`}
             />
             {showLanguages && (
-            <div ref={languageRef} className="overflow-hidden absolute z-50 bottom-[-170px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center">          
-            {/* map languages */}
-            {Object.entries(languages).map(([languageName, languageCode]) => (
-                <button
-                  key={languageCode}
-                  onClick={() => handleLanguageChange(languageCode)}
-                  className={`hover:bg-gray-300 px-5 py-2 w-full ${selectedLanguage === languageCode ? 'bg-secondary-500 py-3 font-bold text-white' : ''}`}
-                >
-                  {t(languageName)}
-                </button>
-              ))}
-              {/* Add more buttons for additional languages */}
-            </div>
-            )} 
+              <div
+                ref={languageRef}
+                className="overflow-hidden absolute z-50 bottom-[-170px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center"
+              >
+                {/* map languages */}
+                {Object.entries(languages).map(
+                  ([languageName, languageCode]) => (
+                    <button
+                      key={languageCode}
+                      onClick={() => handleLanguageChange(languageCode)}
+                      className={` px-5 py-2 w-full 
+                      ${
+                        selectedLanguage === languageCode
```

Comment:
did you change the style of the language list ??

Reply:
Yes, I did as Tam had a screenshot of the selected language and how the font is white and the bg is grey. I fixed it in this PR